core/jobs/batch_jobs/firebase_validation_jobs.py

```
@@ -0,0 +1,145 @@
+# FIXABLE outputs are _expected_, and can be fixed by running `FirebaseSyncJob`.
+TAG_FIXABLE = 'FIXABLE'
+
+# CORRECT outputs are _expected_, and can be used to confirm that things are OK.
+TAG_CORRECT = 'CORRECT'
+
+
+class FirebaseAuditRecordsJob(base_jobs.JobBase):
+    """Audit Firebase records against the records that Oppia claims to exist."""
+
+    def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
+        key_by_email = lambda record: (record.email, record)
```

Ruff (0.15.15)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `key_by_email` as a `def`

(E731)